source/stylesheets/scss/furatto/_alerts.scss

```diff
@@ -0,0 +1,200 @@
+//Alerts
+
+//
+//@variables
+//
+
+@import "compass";
+@import "functions";
+
+//General settings
+$alert-padding: px-to-rems(15) px-to-rems(40) px-to-rems(15) px-to-rems(15) !default;
+$alert-margin-bottom: px-to-rems(20) !default;
+$alert-font-size: px-to-rems(14) !default;
+$alert-font-weight: 300 !default;
+$alert-default-background: #0092FC !default;
+$alert-text-color: rgba(255,255,255,.84) !default;
```

Comment:
`.84` should be written with a leading zero as `0.84`<br>Commas in function arguments should be followed by a single space